.gitlab-ci.yml

```diff
@@ -301,7 +301,7 @@ integration: ## merging all artifacts to produce single R repository, documentat
     R_DEVEL_BIN_VERSION: "4.1"
   script:
     ## pkgdown installs pkgs from "." so run at start to have clean root dir
-    - apt-get update -qq && apt-get install -y libxml2-dev
```

Comment:
could you link the dockerfile if it's public

Reply:
To be used by the the yaml they have to be _in_ the yaml so are you looking for these two (and I am quoting):

- `image: registry.gitlab.com/jangorecki/dockerfiles/r-builder`
- `image: registry.gitlab.com/jangorecki/dockerfiles/r-base-dev`

A top-level `apt` call, however, would be at the level of the CI script / vm, not the container.

Reply:
All images are built and published on GL, see jangorecki/dockerfiles, in menu you will find container registry there